src/app/core/entity-components/entity-subrecord/list-paginator/list-paginator.component.ts

```diff
@@ -92,7 +90,12 @@ export class ListPaginatorComponent<E extends Entity>
     this.updateUserPaginationSettings();
   }
 
-  private applyUserPaginationSettings() {
+  private async applyUserPaginationSettings() {
+    if (!this.user) {
+      const currentUser = this.sessionService.getCurrentUser();
+      this.user = await this.entityMapperService.load(User, currentUser.name);
+    }
```

Comment:
could go into a help method `await ensureUserIsLoaded();` to avoid the duplication below